src/main/java/liquibase/addFhirGlobalLibraryRows.xml

```diff
@@ -7,15 +7,15 @@
+            INSERT INTO CQL_LIBRARY(ID, MEASURE_ID, SET_ID, CQL_NAME, DRAFT, VERSION, FINALIZED_DATE, RELEASE_VERSION, OWNER_ID, LOCKED_USER, LOCKED_OUT_DATE, CQL_XML, REVISION_NUMBER, QDM_VERSION, LAST_MODIFIED_ON, LAST_MODIFIED_BY, LIBRARY_MODEL, FHIR_VERSION) VALUES ('AdultOutpatientEncounters-FHIR4-1-1-000', null, '225b9882-8a50-44a7-88e6-139ad91bfc5c', 'AdultOutpatientEncounters_FHIR4_1_1_000', 1, 0.000, '402803826529d99f0165d450f7d02f61', 'v6.0', '2ca880a17040315101704475bc4f009b', null, null, 0x3C3F786D6C2076657273696F6E3D22312E302220656E636F64696E673D225554462D3822207374616E64616C6F6E653D22796573223F3E3C63716C4C6F6F6B55703E3C6C6962726172793E4164756C744F757470617469656E74456E636F756E746572735F46484952345F315F315F3030303C2F6C6962726172793E3C76657273696F6E3E302E302E3030303C2F76657273696F6E3E3C7573696E674D6F64656C3E51444D3C2F7573696E674D6F64656C3E3C7573696E674D6F64656C56657273696F6E3E352E353C2F7573696E674D6F64656C56657273696F6E3E3C63716C436F6E746578743E50617469656E743C2F63716C436F6E746578743E3C636F646553797374656D732F3E3C76616C7565736574732F3E3C636F6465732F3E3C706172616D65746572733E3C706172616D657465722069643D2266666634326531362D316461312D346566652D613636342D36653762633735386165636522206E616D653D224D6561737572656D656E7420506572696F642220726561644F6E6C793D2274727565223E3C6C6F6769633E496E74657276616C266C743B4461746554696D652667743B3C2F6C6F6769633E3C2F706172616D657465723E3C2F706172616D65746572733E3C646566696E6974696F6E732F3E3C66756E6374696F6E732F3E3C696E636C7564654C696272617279732F3E3C2F63716C4C6F6F6B55703E, 0, '5.5', '2020-03-18 10:24:33', '2ca880a17040315101704475bc4f009b', 'FHIR', null) DUPLICATE KEY UPDATE CQL_NAME=,;
```

Comment:
Could you update the insert statements so that
FHIR_VERSION -> '4.0.0'
QDM_VERSION -> null
DRAFT -> '0'
if that makes sense.
Also, looks like all the CQL_XML has 
<usingModel>QDM</usingModel><usingModelVersion>5.5</usingModelVersion>
But should have  <usingModel>FHIR</usingModel><usingModelVersion>4.0.0</usingModelVersion>
It's causing validation issues

Reply:
I can try to prepare the necessary insert statements. Let me know

Reply:
i got it making the change.

Reply:
Added that but it needs barebones mat xml too. Will work on that in another story. Going to do that next.